src/blueprints/tv2/tv2-action-service.ts

```diff
@@ -84,4 +87,21 @@ export class Tv2ActionService implements BlueprintGenerateActions {
         }
       })
   }
+
+  private getPieceTypeFromMongoActionManifest(mongoActionManifest: ActionManifest): Tv2PieceType {
```

Comment:
It isn't a mongo type at this point.

Reply:
Nicely spotted :) I have removed the mongo references.